packages/schema-extract/README.md

```diff
@@ -0,0 +1,246 @@
+# Schema Extract
+
+
+Schema extract transforms code files into [JSON-Schema](https://json-schema.org/) schemas. Schema extraction is a necessary step to create automated documentation for your code.
+
+It is composed of two parts, the **ts transformer** and **linker**.
+
+
+## Playground
+The [schema-playground](../schema-playground) is a playground that allows live editing of your code and transforms it to a schema and displays the results as you type it.
+
+## Usage
+By default, the linker is connected to our TS transformer, it uses it to transforms the code and then link it together.  
+If you want to use the linker with a different extractor please see the [creating a custom linker section](#creating-a-custom-linker)
+
+Import the `createLinker` function from `@ui-autotools/schema-extract`. This function receives a string array of file paths and returns an initialized linker class.
```

Comment:
"To use the Schema Extract project to extract type data from your files, import the `createLinker`..." I would just make it very clear here that _this_ is how you use Schema Extract and that using the transformer/linker are internal/more specific methods